mirrulations-dashboard/src/mirrdash/dashboard_server.py

```diff
@@ -28,13 +28,18 @@ def get_jobs_stats(database):
     }
 
 
+def get_container_name(container):
+    name = container.name.replace('-', '_')
```

Comment:
Add a comment on this line explaining the `-` vs. `_` issue with Docker.

Reply:
Comment added